src/ToolBox/SOS/lldbplugin/CMakeLists.txt

```diff
@@ -16,7 +16,11 @@ endif (CORECLR_SET_RPATH)
 add_definitions(-DPAL_STDCPP_COMPAT)
 
 set(ENABLE_LLDBPLUGIN ${CLR_CMAKE_PLATFORM_UNIX} CACHE BOOL "Enable building the SOS plugin for LLDB.")
-set(REQUIRE_LLDBPLUGIN ${CLR_CMAKE_PLATFORM_LINUX} CACHE BOOL "Require building the SOS plugin for LLDB.")
+set(REQUIRE_LLDBPLUGIN ${PLATFORM_SUPPORTS_LLDB_PLUGIN} CACHE BOOL "Require building the SOS plugin for LLDB.")
+
+if (${CLR_CMAKE_PLATFORM_LINUX} OR ${CLR_CMAKE_PLATFORM_FREEBSD})
```

Comment:
This needs to be before setting the cache variables and the PLATFORM_SUPPORTS_LLDB_PLUGIN should also be used for the ENABLE_LLDBPLUGIN.